WeatherNotFound/src/main/java/me/learning/weathernotfound/presentation/WeatherNotFound.kt

```diff
@@ -66,7 +68,6 @@ class WeatherNotFound private constructor() {
      */
     fun init(
```

Comment:
We have a problem here: Let's assume I wrote the `auto-init` metadata and set it `true`, after that I called `init` function also in `BaseApplication`. in this scenario, we init library twice!

For this problem, we should throw an exception for developers to understand that they initialized library twice and it is wrong!